packages/react/src/oxygen-ui-react-auth-components/SignInFooter/sign-in-footer.scss

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+.OxygenSignInFooter {
+  width: 100%;
+  margin: 16px 0;
+  color: grey;
+
+  .OxygenSignInLink {
+    color: grey;
```

Comment:
Shall we use a color code?

Reply:
updated with bc928d36f5149ccc0de2bab1afaec3e614eddef3